benefits/eligibility/verify.py

```diff
@@ -21,18 +21,18 @@ def eligibility_from_api(flow: models.EnrollmentFlow, form, agency: models.Trans
         timeout=settings.REQUESTS_TIMEOUT,
     )
 
-    response = client.verify(sub, name, agency.type_names_to_verify(flow))
+    response = client.verify(sub, name, [flow.system_name])
 
     if response.error and any(response.error):
         return None
-    elif any(response.eligibility):
-        return list(response.eligibility)
+    elif [flow.system_name] == response.eligibility:
```

Comment:
This syntax is a little strange... it works, but it might be more natural to say:

`elif flow.system_name in response.eligibility`.

In any case, a comment here about what `response.eligibility` contains would be helpful.

Reply:
I agree, using `in` for this check [looks a lot clearer](https://github.com/cal-itp/benefits/blob/903f76f7bf932cf401dba9e78887ee6b2da5c1b6/benefits/eligibility/verify.py#L29).